pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -73198,7 +73198,6 @@ self: {
        description = "Convert between Dhall and JSON or YAML";
        license = stdenv.lib.licenses.bsd3;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
@mcwitt Thanks for making the effort to fix this.

Unfortunately this file `hackage-packages.nix` is regenerated automatically once a day based on the `configuration-hackage2nix.yaml` file.  You'll have to remove these three packages from the `broken-packages` section in that file.

Reply:
Thanks! I had been confused about which files were auto-generated to Hackage. This makes sense, changed.